NGLess/Interpretation/Count/RefSeqInfoVector.h

```diff
@@ -66,6 +66,6 @@ struct RefSeqInfoVector {
 };
 
 extern "C" {
-    void* rsiv_free(void* p) { delete static_cast<RefSeqInfoVector*>(p); }
+    void rsiv_free(void* p) { delete static_cast<RefSeqInfoVector*>(p); }
```

Comment:
`void*` was just a bug in the previous code, surprising that `G++` accepted it before.

Reply:
Lets see if this fixes the build.